src/plugins/cal-events/cal-events.js

```diff
@@ -168,15 +168,16 @@ var componentName = "wb-calevt",
 					}
 				]
 			},
-			objEventsList = obj.find( "ol > li, ul > li" ),
-			iLen = objEventsList.length,
+			objEventsList = obj.find( "ol, ul" ).first(),
```

Comment:
```suggestion
			objEventsList = obj.find( "ul, ol" ).first(),
```

And apply the same fix on our working example. + Add a note about this.